dev-requirements.txt

```diff
@@ -15,4 +15,4 @@ Flask>=1.1
 # needed for auto fix
 ruff===0.2.2
 # needed for dapr-ext-workflow
-durabletask>=0.1.1a1
+durabletask-dapr >= 0.2.0a3
```

Comment:
FYI it is a terrible decision that this fork made `grpcio` minimum version as high as it did because it impacts interop with lots of older libraries that pinned older versions. There are plenty of Dapr customers using outdated ML libraries that do this.

Reply:
Since the dapr python sdk is the only user of durabletask-dapr, it's not problematic to reevaluate this decision. @famarting , what do you say?